config/dev.json

```diff
@@ -0,0 +1,37 @@
+{
+  "language": "Solidity",
```

Comment:
Can get rid of this file

Reply:
Still need it to test if code compiles and whatnot. I like to keep it because it's not useless.